assets/src/dashboard/app/views/shared/storyGridView.js

```diff
@@ -54,6 +55,14 @@ const StoryGridView = ({
   bottomActionLabel,
 }) => {
   const [contextMenuId, setContextMenuId] = useState(-1);
+
+  const handleMenuItemSelected = useCallback((sender, story) => {
+    setContextMenuId(-1);
+    if (sender.value === STORY_CONTEXT_MENU_ITEMS[0].value) {
```

Comment:
Nit, this isn't very clear what context menu item is being selected 😱.  What if we did the following:
```
// Create a CONTEXT_MENU_ACTIONS enum in constants.js like this:
export const CONTEXT_MENU_ACTIONS = {
  OPEN_IN_EDITOR: 'open-in-editor-action',
  ...,
}

// Update STORY_CONTEXT_MENU_ITEMS constant as follows:
export const STORY_CONTEXT_MENU_ITEMS = [
  {
    label: __('Open in editor', 'web-stories'),
    value: CONTEXT_MENU_ACTIONS.OPEN_IN_EDITOR,
  },
  ...,
]
```
And then we can use `CONTEXT_MENU_ACTIONS` in the if statement like so:
```
if (sender.value === CONTEXT_MENU_ACTIONS.OPEN_IN_EDITOR) {
  ...
}
```
And then it'll be pretty clear which action is being called (in fact, when we hook up the rest, we can just use a `switch` with each `case` being a `CONTEXT_MENU_ACTIONS` item).

Reply:
Good idea.